indigo_app/static/javascript/indigo/views/document_toc.js

```diff
@@ -94,9 +94,26 @@
           qualified_id = $component.attr('eId') + '/' + qualified_id;
         }
 
+        function getHeadingText(node) {
+          var heading = node.children('heading')[0];
```

Comment:
I don't think `ParentNode.children()` accepts parameters: https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children - or is my understanding of `node` incorrect?

I still think xpath is the best way to do this...

Reply:
Oh, node is a jquery object.

Reply:
Perhaps pass in a plain node object to this method, and then locally wrap it in a jquery object so that it's clearer what's going on.